crates/re_types/src/components/origin2d_ext.rs

```diff
@@ -0,0 +1,20 @@
+use crate::datatypes::Vec2D;
+
+use super::Origin2D;
+
+impl Origin2D {
+    pub const ZERO: Self = Self::new(0.0, 0.0);
+
+    #[inline]
+    pub const fn new(x: f32, y: f32) -> Self {
+        Self(Vec2D::new(x, y))
+    }
+}
+
+#[cfg(feature = "glam")]
```

Comment:
This reminds me: we should start using `mint` for this: https://crates.io/crates/mint

Reply:
(not in this PR though)

Reply:
https://github.com/rerun-io/rerun/issues/3292